bundles/org.openhab.ui.start/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,22 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.ui.start
+Bundle-ManifestVersion: 2
+Bundle-Name: openHAB Start UI
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.ui.start
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Import-Package: 
+ javax.servlet,
+ javax.servlet.http,
+ org.apache.commons.io;version="2.2.0",
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.openhab.ui.dashboard,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.component.annotations;resolution:=optional,

```

Comment:
AFAIR this important should not be needed

Reply:
Was copied from ui.dashboard - removed it there as well now